lib/animina/accounts/resources/report.ex

```diff
@@ -169,6 +172,15 @@ defmodule Animina.Accounts.Report do
     prepare build(load: [:accuser, :accused, :admin])
   end
 
+  defp send_notification_emaail_to_admin(id) do
+    UserEmail.send_email(
+      "Stefan Wintermeyer",
+      "stefan@wintermeyer.de",
+      "New Report on Animina",
+      "Hi Stefan\n\nA new report has been submitted on Animina.\n\n-- \nYou can review the report on https://animina.de/admin/reports/pending/#{id}/review"
+    )
```

Comment:
Please add the text of the report in here. So that if I read the email I can already get a feeling of the situation.

Reply:
I have done so , thanks